### PR TITLE
[#2PlaysAMonth]: Wordle Game

### PR DESCRIPTION
**Pull request by SamirMishra27**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Guess the 5-letter word in 6 attempts! A fun word guessing puzzle.

React play submission as mentioned in Issue #1025 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Project was manually tested both on my local system and mobile device to ensure all features are working as well as responsiveness.
# Checklist:

- [x] I have performed a self-review of my own code 
- [x] I have commented my code, particularly in hard-to-understand areas 

# Screenshots or example

Screenshot 2023-03-05 074334
 
Screenshot 2023-03-05 195937

Screenshot 2023-03-03 224619

### CODE REVIEW

File: src/plays/wordle/styles.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
+/* enter stlyes here */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
```
Comment (atapas) [inline src/plays/wordle/styles.css:3]: @SamirMishra27 Can you please use this class name `wordle-game-body` to prefix all the styles in this file?

**Review by atapas — CHANGES_REQUESTED** [review]
@SamirMishra27 

Given a comment...

plus

- Add a cover image
- Add the demo recording.

File: src/plays/wordle/styles.css (hunk 1 above)
Comment (SamirMishra27) [reply src/plays/wordle/styles.css:3]: I'm confused, I will remove `:root selector` from the file?

Comment (SamirMishra27) [reply src/plays/wordle/styles.css:3]: Done!

**Comment by SamirMishra27** [comment]
@atapas Stackstream recording link is already in the project details. It has both coding and demo of project within it. Should I just record a demo only and add that link?

**Review by atapas — APPROVED** [review]

**Pull request merged**